newrelic-weaver/src/main/java/com/newrelic/weave/utils/WeaveUtils.java

```diff
@@ -182,8 +183,8 @@ public final class WeaveUtils {
     private static int getRuntimeMaxSupportedClassVersion() {
         try {
             double jvmSpecVersion = Double.valueOf(System.getProperty("java.specification.version"));
-            if (jvmSpecVersion >=11) {
-                return 55;
+            if (jvmSpecVersion >= 11) {
+                return (int) jvmSpecVersion + CLASS_FILE_VERSION_OFFSET;
```

Comment:
I know down casting isn't great here. Maybe there's a better way. I figured even with Java's new fast release cycle it'll be quite a while until this overflows.  ;-)